Hangman/Hangman/main.swift

```diff
@@ -8,5 +8,238 @@
+
+var randomIndex = Int.random(in: 0..<allTheWords.count)
```

Comment:
Great naming conventions, love the way you ordered your variables and arranged them at the top 👍